deploy/040-controller.yaml

```diff
@@ -54,6 +55,11 @@ spec:
                 configMapKeyRef:
                   name: driver-config
                   key: infraClusterLabels
+            - name: ALLOWED_STORAGE_CLASSES
+              valueFrom:
+                configMapKeyRef:
+                  name: driver-config
+                  key: allowedInfraStorageclasses
```

Comment:
how would you feel about the key `infraStorageClassEnforcement` ?

Reply:
Sounds good, I added it.